translations/messages-en.properties

```diff
@@ -987,3 +987,4 @@ bulkdelete.confirm.action = Delete
 bulkdelete.complete.title = Bulk delete complete
 bulkdelete.complete = Bulk deletion has completed. Click complete to refresh the page.
 bulkdelete.complete.action = Complete
+contact.lastVisitedDate=Visited {{date || 'unknown'}}
```

Comment:
I think this needs whitespace around the `=` to be valid `.properties` format. And I'm not sure that the `{{date || 'unknown'}}` is valid is it? Finally, we don't usually camel case translation keys.

It it were me I would have...

```
contact.last.visited.date = Visited {{date}}
contact.last.visited.unknown = Unknown last visited date
```

Then have a switch in the template to pick between.

Reply:
`{{date || 'unknown'}}` works fine, we already use it elsewhere in our translations (that's how I knew it existed!)

re camelCase, I don't really think we have a convention honestly, I looked through our translations attempting to find one. I'm happy to change the scheme to whatever you feel like.